about_betydb.md

```diff
@@ -65,19 +65,26 @@ All public data in BETYdb is made available under the [Open Data Commons Attribu
 
 If you use or refer to BETYdb data or software in your research, please cite this paper:
 
-> LeBauer D, Kooper R, Mulrooney P, Rohde S, Wang D, Long SP, Dietze MC. BETYdb: a yield, trait, and ecosystem service database applied to second‐generation bioenergy feedstock production. GCB Bioenergy. 2018 Jan;10(1):61-71.  https://doi.org/10.1111/gcbb.12420
+> LeBauer, D., Kooper, R., Mulrooney, P., Rohde, S., Wang, D., Long, S. P., & Dietze, M. C. (2018). BETYdb: a yield, trait, and ecosystem service database applied to second‐generation bioenergy feedstock production. _GCB Bioenergy, 10_, 61–71.  [doi:10.1111/gcbb.12420](https://doi.org/10.1111/gcbb.12420){target="_blank"}
 
-In addition, if you are using **Data** from an instance of BETYdb, please 
+In addition, if you are using _data_ from an instance of BETYdb, please
 
 1. Use the citations associated with each record if the publisher will allow
-2. Refer to the footer of the instance of BETYdb that you are using for citation information. 
+2. Refer to the footer of the instance of BETYdb that you are using for citation information.
    * For example, if you are using data from BETYdb.org, please cite the source of data as:
 
 > LeBauer, David; Dietze, Michael; Kooper, Rob; Long, Steven; Mulrooney, Patrick; Rohde, Gareth Scott; Wang, Dan; (2010): Biofuel Ecophysiological Traits and Yields Database (BETYdb); Energy Biosciences Institute, University of Illinois at Urbana-Champaign. [http://dx.doi.org/10.13012/J8H41PB9](http://dx.doi.org/10.13012/J8H41PB9){target="_blank"}
```

Comment:
The footer of the current version on betydb.org has
> LeBauer, David, Rob Kooper, Patrick Mulrooney, Scott Rohde, Dan Wang, Stephen Long, Michael Dietze, (2017). BETYdb: A yield, trait, and ecosystem service database applied to second‐generation bioenergy feedstock production. GCB Bioenergy. doi:10.1111/gcbb.12420

The citation information given in the example doesn't match this.  But I wasn't quite sure if this information is quite correct either, so I didn't update the example.